packages/theme/src/theme/_navs.scss

```diff
@@ -73,41 +73,42 @@
 	}
 
 	&-tabs {
+		background-color: tint($brand-primary, 90);
```

Comment:
I don't understand what for we need this variable. It's used only in one place. It shouldn't be rewritten. It's not rewrite bootstrap variable. Could you please explain?